src/resources/ha-style.ts

```diff
@@ -57,7 +57,7 @@ const mainStyles = css`
     --label-badge-grey: #9e9e9e;
 
     /* states icon */
-    --state-icon-color: #44739e;
+    --state-icon-color: var(--paper-item-icon-color, #44739e);
```

Comment:
This will still be breaking, as it only applied from the top theme and not elements that are individual themed.

Reply:
I don't necessarily think that is a problem though

Reply:
I've tried already many things, but is there a possibility to solve this without breaking change? 
If not, we will need to collect these changes to when Polymer is removed and do them at once.

Reply:
If we change all the uses of `state-icon-color` to have a default, like:
`var(--state-icon-color, #44739e);`, remove it from `ha-style` and put `state-icon-color`: `paper-item-icon-color` in derived styles, I _THINK_ it should not break anything, but not 100% sure...